datastream-server/src/main/java/com/linkedin/datastream/server/EventProducer.java

```diff
@@ -286,6 +325,10 @@ private void reportMetrics(DatastreamRecordMetadata metadata, long eventsSourceT
       _dynamicMetricsManager.createOrUpdateCounter(MODULE, AGGREGATE, TOTAL_EVENTS_PRODUCED, 1);
       _dynamicMetricsManager.createOrUpdateCounter(MODULE, _datastreamTask.getConnectorType(), TOTAL_EVENTS_PRODUCED,
           1);
+
+      // Log information about events if either warn logging is enabled or logging for topic partitions outside
+      // alternative SLA is enabled
+      performSlaRelatedLogging(metadata, eventsSourceTimestamp, sourceToDestinationLatencyMs);
```

Comment:
is the move inside of _logger.isDebugEnabled() intentional?

Reply:
this is not within the if block, the diff just swallowed the rest of the if block.

Reply:
Ah sorry, got confused with the indent.